site/content/docs/5.0/migration.md

```diff
@@ -9,6 +9,10 @@ toc: true
 
 ## v5.0.0-alpha3
 
+### Browser support
+
+- Dropped support for Microsoft Legacy Edge. See [here](#browser-support-1) for the previous browser support changes.
```

Comment:
Not sure if there is any other way to do this so that it's foolproof; now it's fragile and will break if we add another `Browser support` heading.